index.html

```diff
@@ -196,69 +197,77 @@ <h3 class="product-card-title">Бабл-гам</h3>
           </li>
         </ul>
       </section>
+      <div class="benefits">
       <section class="advantages">
         <h2 class="advantages-title">Магазин Глейси – это онлайн и офлайн-магазин по продаже мороженого собственного производства на развес</h2>
         <ul class="advantages-list">
-          <li class="advantages-item basic-text">
+          <li class="advantages-item-ice-cream basic-text">
```

Comment:
А почему вот эти классы ты добавила к основному: `<li class="gifts-item gifts-item-raspberry">`
а тут заменила основной: `<li class="advantages-item-ice-cream basic-text">` ?

Лучше всегда добавлять, это удобно: основные стили через общий класс, а картинки через дополнительный можно будет ставить:
```suggestion
          <li class="advantages-item advantages-item-ice-cream basic-text">
```

Reply:
поправила